package.json

```diff
@@ -79,7 +79,7 @@
     "proj4": "2.4.4",
     "prop-types": "15.6.1",
     "query-string": "5.1.1",
-    "react-dom": "16.3.2",
+    "react-dom": "16.4.0",
```

Comment:
This would be done by greenkeeper, wouldn't it?

Reply:
The need of this PR was (amongst others) introduced by the `react-dom` update. Since the changes made here are fixing the failing build of the [original](https://github.com/terrestris/react-geo/pull/665) greenkeeper PR, it should be fine to have it updated herein.